src/main/java/picard/util/IntervalListTools.java

```diff
@@ -513,6 +519,9 @@ private IntervalList openIntervalLists(final List<File> files, BinaryOperator<In
         if (COUNT_OUTPUT != null && OUTPUT_VALUE == Output.NONE) {
             errorMsgs.add("COUNT_OUTPUT was provided but OUTPUT_VALUE is set to NONE.");
         }
+        if (ACTION == Action.UNION && DONT_COMBINE_ABUTTING) {
+            errorMsgs.add("ACTION=UNION action combines abutting intervals by default, so it is incompatible with DONT_COMBINE_ABUTTING=true.");
```

Comment:
I'm not sure I understand why this is inherently incompatible with UNION. I get that _currently_ the htsjdk `uniqued` method combines abbutting intervals, but I don't see why it has to be that way....it seems plausible, to work around it by using the lower-level `getUniqueIntervals` directly.

Reply:
There is an additional wrinkle, that `getUniqueIntervals` returns `List<Intervals>` whereas `uniqued()` returns `IntervalList`, and there is no constructor that takes in a `List<Intervals>` and turns it into an `IntervalList`. I can see if there's a way to do this with `addAll()` and removing the old intervals. If you have any ideas let me know.

Reply:
you can replicate the htsjdk code to make a private method that also allows you to state "mergeAbutting", right? eventually perhaps someone will push that down to htsdjk.

Reply:
Fixed @yfarjoun